packages/opticss/package.json

```diff
@@ -11,8 +11,7 @@
     "lint": "tslint -t msbuild --project tsconfig.json -c tslint.cli.json",
     "lintall": "tslint -t msbuild --project tsconfig.json -c tslint.release.json",
     "lintfix": "tslint -t msbuild --project tsconfig.json -c tslint.cli.json --fix",
-    "test": "mocha dist/test/**/*.js --opts test/mocha.opts -i --grep Integration",
-    "integration": "yarn run pretest && mocha dist/test --opts test/mocha.opts --grep Integration",
+    "test": "mocha dist/test --opts test/mocha.opts",
```

Comment:
It didn't seem useful to me to separate integration tests from the rest – just made it confusing coming in with fresh eyes and wondering why some tests weren't running! Happy to change it back if you feel strongly.

Reply:
The integration tests are slow and I expected them to only get slower. We can extract them back out later if it gets onerous.